### PR TITLE
Clarify how to be added to allow list

### PR DESCRIPTION
**Pull request by Arthur-Milchior**
On first reading, I found very confusing to understand how I could be added to
allow list. I just tried creating the PR anyway and discovered that the first
message by your bot explains how to add someone. I believe that it would improve
first user interaction if they already knew how to find this user that may allow
them. (Even better would be to have it automated, but that's a more complex task
I don't dare to try to work on)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -113,7 +113,8 @@ <h2>How can you use GitGitGadget?</h2>
 				You might also want to read the <a href="https://git-scm.com/docs/gitworkflows">gitworkflows</a> manual page to understand how your contributions will be integrated in Git's repository, as well as <a href="https://github.com/git/git/blob/todo/MaintNotes">this note from Git's maintainer</a>.
 			</p><p>
 				The first time you use GitGitGadget, you need to be added to the list of users with permission to use GitGitGadget (this is a <i>very</i> simple anti-spam measure).
-				Any user who is already on that list can do that, by adding a comment to that Pull Request that says <code>/allow &lt;username&gt;</name></code> (with your GitHub login name).
+				Using <a src="https://web.libera.chat/#git-devel">IRC</a>, <a src="https://github.com/gitgitgadget/git/pulls?utf8=%E2%9C%93&q=is%3Apr+is%3Aopen+%22%2Fallow%22">PRs</a> you can find a user already on this list and request them to add you to the list.
```
Comment (dscho) [inline index.html:116]: should the comma be replaced by "or"?

File: index.html [hunk 2]
```diff
@@ -113,7 +113,8 @@ <h2>How can you use GitGitGadget?</h2>
...
+				To do so, they simply have to add a comment to that Pull Request that says <code>/allow &lt;username&gt;</name></code> (with your GitHub login name).
```
Comment (dscho) [inline index.html:117]: The preferred form is `/allow` (without username) on the PR opened by the user who should be added to the allow list.

**Review by dscho — CHANGES_REQUESTED** [review]
It is very exciting to see how you improve the documentation to make it easier to contribute to Git. Thank you!

**Comment by dscho** [comment]
@Arthur-Milchior Are you still interested in this PR?

File: index.html (hunk 1 above)
Comment (Arthur-Milchior) [reply index.html:116]: done

File: index.html (hunk 2 above)
Comment (Arthur-Milchior) [reply index.html:117]: Thanks for letting me know. I was trying to rephrase the content I understood and didn't know it was not up to date

**Review by Arthur-Milchior — COMMENTED** [review]
Thanks for pinging.
I fear I have let more than a thousand open notification in github, and so sometime some actual important ones fails through.
Correction done.

**Pull request closed without merging**